src/components/LandingNav/LandingNav.js

```diff
@@ -17,11 +18,20 @@ const LandingNav = ({ buttons, title, position }) => {
               <img src={button.iconSrc} alt='' />
               <div className={styles['button-name']}>{button.name}</div>
             </a>
-          ) : (
+          ) : button.name !== 'Play Dice' ? (
```

Comment:
Having conditionals complicates the component.  I think it would've been easier to just declare them statically, not via buttons prop